README.md

```diff
@@ -25,9 +48,9 @@ $ echo "aws-go-fargate" >> .git/info/sparse-checkout ## update this
 $ git pull origin master
 ```
 
-Don't see an example listed? [Try Pulumi AI](https://www.pulumi.com/ai) and use natural-language prompts to generate Pulumi infrastructure-as-code programs in _any_ language.
+Don't see an example listed? [Try Pulumi Copilot](https://app.pulumi.com/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=pulumi-examples) and use natural-language prompts to generate Pulumi infrastructure-as-code programs in _any_ language.
 
-## Outline
+## Example Library
```

Comment:
Maybe it's me, but "list" seems like a clearer term here than "library". "All examples" could also work. (For some reason when I read "library" and "go back to the library", I kept getting the feeling I was being sent somewhere else, until I realized it was just to another position on this same page.) 

```suggestion
## All examples
```